### PR TITLE
poolManager.RequestContainerV5: Integer constants replaced by enum called RequestStatusCode

### PR DESCRIPTION
**Pull request by dcachess17**
Motivation: Prior to Java 1.5 enums did not exist, thus integer constants were used. These constants can now be replaced by enums.

Modification: diskCacheV111/poolManager/RequestContainerV5  return code constants were replaced by an enum called RequestStatusCode
for instance :    RT_OK  to  RequestStatusCode.OK

Result: slightly higher memory consumption for the benefits of type safety and more maintainable code

Signed-off-by: Vuong Luu Minh s0548844@htw-berlin.de

### CODE REVIEW

File: modules/dcache/src/main/java/diskCacheV111/poolManager/RequestContainerV5.java [hunk 1]
```diff
@@ -1080,15 +1094,7 @@ private boolean answerRequest(int count) {
...
-        private static final int RT_NOT_FOUND  = 3 ;
-        private static final int RT_ERROR      = 4 ;
-        private static final int RT_OUT_OF_RESOURCES = 5 ;
-        private static final int RT_COST_EXCEEDED    = 7 ;
-        private static final int RT_NOT_PERMITTED    = 8 ;
-        private static final int RT_S_COST_EXCEEDED  = 9 ;
-        private static final int RT_DELAY  = 10 ;
+
 
         private static final int CONTINUE        = 0 ;
```
Comment (alrossi) [inline modules/dcache/src/main/java/diskCacheV111/poolManager/RequestContainerV5.java]: The way the state machine is implemented here of course could be rewritten, which is not the task at hand, but I am wondering whether a package-visible enum for these two states might not also be desirable.

**Comment by kofemann** [comment]
ok to test

**Comment by alrossi** [comment]
It's fine.

**Comment by kofemann** [comment]
Thanks!

**Pull request merged**